samples/build.gradle

```diff
@@ -5,13 +5,13 @@ sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
-  compile project(":services-core")
-  compile project(":services-directions")
-  compile project(":services-geocoding")
-  compile project(":services-optimization")
-  compile project(":services-geojson")
-  compile project(":services-matching")
-  compile project(":services-staticmap")
+  implementation project(":services-geocoding")
+  implementation project(":services-optimization")
+  implementation project(":services-geojson")
+  implementation project(":services-matching")
+  implementation project(":services-staticmap")
+
+  implementation dependenciesList.retrofit
```

Comment:
Why is this needed for the sample project, doesn't one of our service libraries carry retrofit over?

Reply:
@tobrun @cammace I ran into this this as well when I was doing initial changes and had to include `implementation dependenciesList.retrofit` but then eventually was able to build without it. I will give a try.